### PR TITLE
Read secrets from GCS in installation script

### PR DESCRIPTION
**Pull request by jakkab**
**Description**

Changes proposed in this pull request:
- Read secrets from GCS in installation script

**Related issue(s)**
#26

### CODE REVIEW

File: prow-configurations/create-gcp-secrets.sh [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    ENCRYPTED_FILE="${FILE}.${EXTENSTION}"
+    gsutil cp gs://${BUCKET}/${ENCRYPTED_FILE} ${TMP_DIR}/${ENCRYPTED_FILE}
+    gcloud kms decrypt --location "${LOCATION}" --keyring "${KEYRING}" --key "${KEY}" --ciphertext-file "${TMP_DIR}/${ENCRYPTED_FILE}" --plaintext-file "${TMP_DIR}/${FILE}"
+    rm ${TMP_DIR}/${ENCRYPTED_FILE}
+done
+
+##########
+
+if [ "$(ls -A ${TMP_DIR})" ]; then
+    kubectl create secret generic ${SECRET_NAME} --from-file=${TMP_DIR}
```
Comment (strekm) [inline prow-configurations/create-gcp-secrets.sh]: Will we have one secret for all files? I guess I would prefer to have one secret per one file stored in GCS. I can imagine having a file for slack token and for victorops token and I don't want to have them both in one k8s secret cause I might have 2 prow jobs: one for integration with slack and another for victorops.

File: prow-configurations/create-gcp-secrets.sh [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+set -o errexit
+
+usage () {
+    echo "Provide correct GCP bucket name, keyring and encryption key!"
+    exit 1
+}
+
+CURRENT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+SECRET_NAME="gcp-prow"
+FILES=("sa-gke-kyma-integration")
```
Comment (strekm) [inline prow-configurations/create-gcp-secrets.sh]: Why not create secrets with file names?

File: prow-configurations/create-gcp-secrets.sh [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+EXTENSTION="encrypted"
```
Comment (strekm) [inline prow-configurations/create-gcp-secrets.sh]: Didn't we agree that files in GCS won't have extensions?

File: prow-configurations/create-gcp-secrets.sh [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+##########
+
+TMP_DIR=`mktemp -d "${CURRENT_DIR}/temp-XXXXXXXXXX"`
+trap "rm -rf ${TMP_DIR}" EXIT
+
+for FILE in "${FILES[@]}"
+do
+    ENCRYPTED_FILE="${FILE}.${EXTENSTION}"
+    gsutil cp gs://${BUCKET}/${ENCRYPTED_FILE} ${TMP_DIR}/${ENCRYPTED_FILE}
```
Comment (strekm) [inline prow-configurations/create-gcp-secrets.sh]: What will happen if the file doesn't exist in the bucket?

File: prow-configurations/create-gcp-secrets.sh (hunk 1 above)
Comment (jakkab) [reply prow-configurations/create-gcp-secrets.sh]: good point, will be changed.

File: prow-configurations/create-gcp-secrets.sh (hunk 2 above)
Comment (jakkab) [reply prow-configurations/create-gcp-secrets.sh]: will be addressed.

**Pull request merged**